agent/segment_stub.go

```diff
@@ -2,18 +2,14 @@
 
 package agent
 
-import (
-	"github.com/hashicorp/consul/agent/structs"
-)
-
-func ValidateSegments(conf *Config) error {
-	if conf.Segment != "" {
-		return structs.ErrSegmentsNotSupported
-	}
-
-	if len(conf.Segments) > 0 {
-		return structs.ErrSegmentsNotSupported
-	}
-
-	return nil
-}
+// done(fs): func ValidateSegments(conf *Config) error {
```

Comment:
Did this get moved elsewhere? There's an enterprise version of this file that's in the enterprise fork, so we had this out here so we can swap it with build tags.

Reply:
I had incorporated this into the normal validation. However, to support different types of validation for enterprise I've split up the tests so that we can overload them. I'll send you the `ent` fragments you can add.